tools/diffbind/diffbind.xml

```diff
@@ -1,11 +1,9 @@
-<tool id="diffbind" name="DiffBind" version="2.1.0">
+<tool id="diffbind" name="DiffBind" version="2.1.16.3">
     <description> differential binding analysis of ChIP-Seq peak data</description>
     <requirements>
-        <requirement type="binary">Rscript</requirement>
-        <requirement type="R-module">DESeq2</requirement>
-        <requirement type="package" version="3.0.3">R</requirement>
-        <requirement type="package" version="1.2.10">deseq2</requirement>
-        <requirement type="package" version="1.8.3">diffbind</requirement>
+        <requirement type="package" version="3.3.2">r</requirement>
+        <requirement type="package" version="1.12.4">deseq2</requirement>
```

Comment:
Deseq2 seems already include within the bioconductor-diffbind onda package. So you can omit it :)